datasets/swda/swda.py

```diff
@@ -83,6 +82,305 @@
 # Dataset main url.
 _URL = "https://github.com/cgpotts/swda/raw/master/swda.zip"
 
+# Dialogue act tags - long version 217 dialogue acts labels.
+act_tags = [
```

Comment:
```suggestion
_ACT_TAGS = [
```

To be consistent with other global variables